test/lowlevel.test.ts

```diff
@@ -181,7 +181,7 @@ describe("lowlevel", function () {
         (err: any) => {
           strictEqual(
             err.message,
-            "validating query: unable to parse the query: syntax error at position 18 near '3.14'"
+            "validating read query: unable to parse the query: syntax error at position 18 near '3.14'"
```

Comment:
There was an update to error messaging that comes back from the Validator, so we have update the tests here.